_posts/2016-12-07-forget-the-keyboard.md

```diff
@@ -22,6 +22,10 @@ Besides the unavoidable and boring hours of exercise, the trick that helped me t
 >
 > Only focus on *which finger to use*.
 
+*What do you mean?*
```

Comment:
Not convinced. 

TL;DR stands for "Too long, don't read", and it is intended to be like a sort of abstract. Just like the "occhiello" o "sommario" in an article.

I would remove that.

Proposal: we could use a different typeset. See for example this [article](https://blog.medium.com/blogging-on-medium-95f1546bcd7d#.4yv4yam3a) (I just took a random one). It uses *italic* font to distinguish the TL;DR section from the rest of the post.

I'm merging this PR, and later I think I will revert this commit, ok?